charmcraft/const.py

```diff
@@ -48,6 +49,31 @@
 MANDATORY_HOOK_NAMES = frozenset(("install", "start", "upgrade-charm"))
 HOOKS_DIRNAME = "hooks"
 
+SUPPORTED_BASES = frozenset(
+    (
+        BaseName("ubuntu", "18.04"),
```

Comment:
out of curiosity, is there a plan to drop bionic?

Reply:
I'm sure there is, but I'm not aware of it right now. I think we have a few big customers using ESR for their charms, so we'd have to be careful.